src/components/Content/Pagination.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Pagination = () => (
+  <div className="row">
+    <div className="col-full">
+      <nav className="pgn">
+        <ul>
+          <li><a className="pgn__prev" href="#0">Prev</a></li>
```

Comment:
Can we use a loop for generating these `li` elements?

Reply:
Do you think it would be better to handle the whole list by js or just the ones which they are similar like numbers? Cause there are other types like the "previous" and "next" and the three-dot in the middle and the list.

Reply:
```suggestion
    const list = () => {
      const pages = [];
      pages.push(<li><a className="pgn__prev" href="#0">Prev</a></li>);
      for (let i = 1; i <= 5; i += 1) {
        pages.push(<li><a className="pgn__num" href="#0">{i}</a></li>);
      }
      pages.push(<li><span className="pgn__num dots">...</span></li>);
      pages.push(<li><a className="pgn__num" href="#0">8</a></li>);
      pages.push(<li><a className="pgn__next" href="#0">Next</a></li>);
    return pages;
  };
```

what do you think of this?